portfolio/src/main/webapp/index.html

```diff
@@ -80,6 +80,6 @@ <h4 class="col-6 col-md-3"><a href="https://github.com/zhangshyue">Github</a></h
         </div>
         <hr>
     </div>
-    <script src="script.js"></script>
+    <!-- <script src="script.js"></script> -->
```

Comment:
Did you comment this out by mistake?

Reply:
No actually, since script.js is already included at the top of index.html.